.github/workflows/format-website-on-dispatch.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
       - name: Set up Node.js
-        uses: actions/setup-node@v4.4
+        uses: actions/setup-node@v4
```

Comment:
this seems like a potential typo?

Reply:
https://github.com/actions/setup-node/releases/tag/v4.4.0 🤔

Reply:
Looks like 4.4 itself isn't a tag, it's either 4.4.0 or 4 that's accepted

Also we use 4 everywhere else so this is more consistent

<img width="1212" alt="image" src="https://github.com/user-attachments/assets/40c50de5-a98f-41fb-8f2e-58cca60a67e8" />

<img width="495" alt="image" src="https://github.com/user-attachments/assets/c97033b6-1c47-4d6e-b705-1577350a8f53" />

<img width="1157" alt="image" src="https://github.com/user-attachments/assets/d71b5452-108b-4ce3-aba7-2a18393cc091" />

Reply:
sorry to be annoying but this is changing website code can we put it in a different PR - this one is so bug we don't want it to be lost

Reply:
done